packages/malloy-db-test/src/datetimes.spec.ts

```diff
@@ -201,6 +201,13 @@ runtimes.runtimeMap.forEach((runtime, databaseName) => {
       });
     });
 
+    describe(`now - ${databaseName}`, () => {
+      test(`generate the current timestamp - ${databaseName}`, async () => {
+        const eq = sqlEq("now + 30 days - 30 days", "now");
+        expect(await eq).isSqlEq();
+      });
+    });
```

Comment:
(Kind of a bad test, but I wanted to avoid introducing a need to freeze the current time in any way. Open to suggestions.)

Reply:
Clever :)  I like it.